File: Platform.Numbers.Tests/MathTests.cs
using Xunit;

namespace Platform.Numbers.Tests
{
    public class MathTests
    {
        [Fact]
        public void CompiledOperationsTest()
        {
            Assert.True(Math.Abs(Arithmetic<double>.Subtract(3D, 2D) - 1D) < 0.01);
        }
    }
}

File: cpp/Platform.Numbers.Tests/MathTests.cpp
namespace Platform::Numbers::Tests
{
    TEST_CLASS(MathTests)
    {
        public: TEST_METHOD(CompiledOperationsTest)
        {
            Assert::IsTrue(Math.Abs(Arithmetic<double>.Subtract(3D, 2D) - 1D) < 0.01);
        }
    };
}
